templates/core/terraform/airlock/airlock_processor.tf

```diff
@@ -0,0 +1,64 @@
+data "local_file" "airlock_processor_version" {
+  filename = "${path.root}/../../../airlock_processor/_version.py"
+}
+
+locals {
+  version = replace(replace(replace(data.local_file.airlock_processor_version.content, "__version__ = \"", ""), "\"", ""), "\n", "")
+}
+
+# re-using the id we have to access the acr
+data "azurerm_user_assigned_identity" "id" {
+  name                = "id-api-${var.tre_id}"
+  resource_group_name = var.resource_group_name
+}
```

Comment:
I think we should have a new one for the airlock as it will need specific permissions not relevant for the API